packaging/freebsd/README.md

```diff
@@ -0,0 +1,22 @@
+mackerel-agent for FreeBSD
+==========================
+
+1. Get binary for FreeBSD at release page. https://github.com/mackerelio/mackerel-agent/releases
+    - `amd64`: `mackerel-agent_freebsd_amd64.tar.gz`
+    - `i386` : `mackerel-agent_freebsd_386.tar.gz`
+    - (`arm` : `mackerel-agent_freebsd_arm.tar.gz`)
+2. Extract file, and `cd`.
+    - `tar -xzvf mackerel-agent_freebsd_*.tar.gz`
+3. Copy `mackerel-agent` to `/usr/local/bin/`.
+    - `(sudo) cp mackerel-agent /usr/local/bin/`
+4. Edit `mackerel-agent.conf`, after copy to `/usr/local/etc/`.
+    - `${EDITOR} mackerel-agent.conf`
```

Comment:
[nit]
Users may use `mackerel-agent.sample.conf` as template?

Reply:
I unarchive released file (e.g. `mackerel-agent_freebsd_amd64.tar.gz`). but it is not included `mackerel-agent.sample.conf`.
So I wrote not `mackerel-agent.sample.conf` but `mackerel-agent.conf`.

```
$ tar xzvf mackerel-agent_freebsd_amd64.tar.gz 
x mackerel-agent_freebsd_amd64/
x mackerel-agent_freebsd_amd64/CHANGELOG.md
x mackerel-agent_freebsd_amd64/LICENSE
x mackerel-agent_freebsd_amd64/LICENSE.rtf
x mackerel-agent_freebsd_amd64/README.md
x mackerel-agent_freebsd_amd64/mackerel-agent
x mackerel-agent_freebsd_amd64/mackerel-agent.conf
```

Reply:
Oh, sorry for misunderstand. Okay.